Hologram-Data-Availability/hda-web-server/WORKSPACE

```diff
@@ -0,0 +1,65 @@
+workspace(name = "cpp_server")
```

Comment:
Change the folder structure into

/hologram_tools
  /server
  /front_end

Reply:
/hologram_tools
->/server
->/front_end

Reply:
/hologram_data_availability
BUILD
->/server
   -> server_main.cc
->/front_end